cmake/generate-cpp.cmake

```diff
@@ -1,4 +1,7 @@
-cmake_minimum_required (VERSION 3.2)
+if (POLICY CMP0007)
+  # Do not ignoere empty list items
```

Comment:
ignoere

Reply:
Changed to `list command no longer ignores empty elements` in rebased commit.